connector/saml/saml.go

```diff
@@ -366,6 +366,8 @@ func (p *provider) HandlePOST(s connector.Scopes, samlResponse, inResponseTo str
 		return ident, fmt.Errorf("response did not contain a AttributeStatement")
 	}
 
+	p.logger.Infof("parsed and verified saml response attributes %s", attributes)
```

Comment:
Does this just display all the attributes together? Could we display this as emailAttr:%s, usernameAttr:%s and groupsAttr:%s?